CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
   and
   [`@digitalbazaar/credentials-examples-context`](https://github.com/digitalbazaar/credentials-examples-context).
 
+### Changed
+- Update dev dependencies.
+- Update `jsonld` and to `@digitalbazaar/ed25519-signature-2018@4`.
```

Comment:
It looks like there is some typo here that needs to be fixed. Also, `@digitalbazaar/ed25519-signature-2018@4` is only a dev dep used in tests. So it can be moved under `update dev dependencies`.